Detections/MultipleDataSources/UnusualGuestActivity.yaml

```diff
@@ -0,0 +1,51 @@
+id: acc4c247-aaf7-494b-b5da-17f18863878a
+name: Unusual guest activity - Inviting external guest user followed by Azure AD powershell signin activity

```

Comment:
Let's shorten the name a bit and match the description a bit closer, remove the part before hyphen and maybe go with "External guest invitation by default guest followed by unusual PowerShell sign in".  Thoughts?

Reply:
made changes. added explicit Azure AD powershell instead just powershell